packages/sitecore-jss-nextjs/src/utils/index.ts

```diff
@@ -0,0 +1,2 @@
+export { getPublicUrl, handleEditorFastRefresh, getJssEditingSecret } from './utils';
```

Comment:
We do export SiteInfo and SiteResolver in two places currently - why doing this with handleEditorFastRefresh, getPublicUrl is a bad idea?
I understand we need our exports to be more predictable. But that would mean we need these functions be exported from /utils not from the root. If we export them from both places this would be a middle ground between predictability and compatibility.

Reply:
Do we? I only see SiteInfo and SiteResolver exported from /middleware currently.

But I see your point and agree it'd be better to have them together under /utils. How about duplicate exports for now, update our template to use /utils exports, and add a @deprecated jsdoc tag to the root exports?

Reply:
Okay, done, please take a look.